bin/upload_new_release_to_s3.py

```diff
@@ -35,11 +35,11 @@ def recursive_upload(dir = None, parent_dirs=None):
     f_path = os.path.join(full_path, f)
     if os.path.isfile(f_path):
       if new_parent_dirs:
-        print(f"uploading {f_path} to s3:cdo-ai/releases/{args.release_name}-{now}/{new_parent_dirs}/{f}")
-        s3.upload_file(f_path, 'cdo-ai', f"teaching_assistant/releases/{args.release_name}-{now}/{new_parent_dirs}/{f}")
+        print(f"uploading {f_path} to s3:cdo-ai/teaching_assistant/releases/{now}-{args.release_name}/{new_parent_dirs}/{f}")
+        s3.upload_file(f_path, 'cdo-ai', f"teaching_assistant/experiments/{now}-{args.release_name}/{new_parent_dirs}/{f}")
       else:
-        print(f"uploading {f_path} to s3:cdo-ai/releases/{args.release_name}-{now}/{f}")
-        s3.upload_file(f_path, 'cdo-ai', f"teaching_assistant/releases/{args.release_name}-{now}/{f}")
+        print(f"uploading {f_path} to s3:cdo-ai/teaching_assistant/releases/{now}-{args.release_name}/{f}")
+        s3.upload_file(f_path, 'cdo-ai', f"teaching_assistant/experiments/{now}-{args.release_name}/{f}")
```

Comment:
This was the version of the script that I ran to generate the output in the PR comments. It posted the aitt_release_data contents to teaching_assistant/experiments/2024-08-07-claude-test in the cdo-ai S3 bucket.